src/daft-schema/src/field.rs

```diff
@@ -1,17 +1,18 @@
-use std::fmt::{Display, Formatter, Result};
 use std::hash::Hash;
 use std::sync::Arc;
 
 use arrow2::datatypes::Field as ArrowField;
 
 use crate::dtype::DataType;
 use common_error::{DaftError, DaftResult};
+use derive_more::Display;
 
 use serde::{Deserialize, Serialize};
 
 pub type Metadata = std::collections::BTreeMap<String, String>;
 
-#[derive(Clone, Debug, Eq, Deserialize, Serialize)]
+#[derive(Clone, Display, Debug, Eq, Deserialize, Serialize)]
+#[display("{name}: {dtype}")]
```

Comment:
the previous I believe was `write!(f, "{}#{}", self.name, self.dtype)`

Let's preserve what the users expect

Reply:
Addressed in follow-up PR:
https://github.com/Eventual-Inc/Daft/pull/2820